app/notification/magic_link/map_contents.py

```diff
@@ -23,8 +28,16 @@ def from_json(json_data: dict):
         content = MagicLink.process_data(json_data).get("content")
         return MagicLink(
             contact_info=data.contact_info,
-            fund_name=content.get("fund_name"),
-            magic_link=data.content.get("magic_link_url"),
+            fund_name=content.get(Config.NOTIFICATION_MAGIC_LINK_FUND_NAME),
+            magic_link=content.get(
+                Config.NOTIFICATION_MAGIC_LINK_MAGIC_LINK_URL
```

Comment:
Should we do these as hard lookups, so we throw if they're not present, rather than returning `None`?

Reply:
We already have a validation (key error) for this & returns an error with expected magic link contents with null value if any of the required key is missing

Reply:
Discussed and we are going to rely on notify returning us an error in that case.